Modify deleteFromDocument description (#35603)
mdn/content: files/en-us/web/api/selection/deletefromdocument/index.md

{{ ApiRef("DOM") }}

The **`deleteFromDocument()`** method of the

> {{domxref("Selection")}} interface deletes the selected {{ domxref("Node", "Nodes") }} from the document's DOM.

Reviewer: ```suggestion
{{domxref("Selection")}} interface deletes the selected {{domxref("Node")}} objects from the document's DOM.
```

`Nodes` is not a code entity. `domxref` is going to code-format it. Let's just reword it instead.

## Examples

> This example lets you delete selected nodes by clicking a button. Upon clicking the

Reviewer: ```suggestion
This example lets you delete selected text and/or nodes by clicking a button. Upon clicking the
```

button, the

> {{domxref("Window.getSelection()")}} method gets the selected objects, and the

Reviewer: ```suggestion
{{domxref("Window.getSelection()")}} method gets the selected content, and the
```

`deleteFromDocument()` method removes it.

### HTML

```html
<p>Try highlighting some of the elements in these paragraphs.</p>
<p>
  Once you do, you can remove the selected content by clicking the button below.
</p>

> <h2>All of the selected nodes are removed.</h2>

Reviewer: ```suggestion
<h2>All of the selected text/nodes are removed.</h2>
```

> <h2>All of the selected text/nodes are removed.</h2>

Reviewer: ```suggestion
<h2>All of the selected text/nodes will be removed.</h2>
```
